### PR TITLE
Use msgpack for returned data

### PR DESCRIPTION
**Pull request by hongthaiphi**

### CODE REVIEW

File: icetea/helper/utils.js [hunk 1]
```diff
@@ -66,15 +77,23 @@ exports.emitEvent = function (emitter, tags, eventName, eventData, indexes = [])
...
       throw new Error("Event's indexed key must be string")
     }
-    // if (typeof eventData[indexedKey] === "undefined") {
-    //    throw new Error("Event's indexed value is not provided");
-    // }
-    tags[eventName + EVENTNAME_INDEX_SEP + indexedKey] = String(JSON.stringify(eventData[indexedKey]))
+    if (typeof eventData[indexedKey] === 'object') {
+      throw new Error("Event's indexed value cannot be an object.")
+    }
+    tags[eventName + EVENTNAME_INDEX_SEP + indexedKey] = String(eventData[indexedKey] == null ? '' : eventData[indexedKey])
```
Comment (thith) [inline icetea/helper/utils.js]: On the right side of = should be:
```js
eventData[indexedKey] == null ? '' : String(eventData[indexedKey])
```

**Comment by thith** [comment]
Would you mind fixing the test cases to make them pass?

**Comment by thith** [comment]
Would you mind create an issue corresponding to this PR?

**Comment by hongthaiphi** [comment]
> Would you mind create an issue corresponding to this PR?

This is the issue #130

**Comment by thith** [comment]
The web for devtools fails. Could you update the web also?

**Comment by thith** [comment]
The test fails somehow, but it runs locally on my machine, so it might be a travis config problem. Will investigate later.

**Pull request merged**